### DISCUSSION TITLE
Can Turborepo be used for a monorepo that also includes a Django server package?

### DISCUSSION THREAD

**Discussion by thedanchez** [vercel/turborepo#1077] [Help] [answered]
Just came across Turborepo recently and had a question as I'm about to build a monorepo for an application. One of the packages of the monorepo I want to make is a Python/Django API server. Would it still be possible to use Turborepo in this case? Or is the expectation such that all the packages have to be JS/TS?

**Comment by drummerjolev** [comment]
Any updates on this? I'm also highly interested in this use case.

A Django/TS monorepo I found in the wild: https://github.com/PostHog/posthog

**Comment by mehulkar** [comment, marked as answer]
You can certainly have a Django app in your monorepo without any issues. If you want Turborepo to _do_ anything with the Django project, you will need to:

- Make sure it is part of your workspaces declaration (e.g. where you declare npm workspaces at the root of your monorepo)
- Have a `package.json` in your DJango app directory with the `name` and `scripts` property

Turborepo does use Node.js conventions to find workspaces and execute tasks, but it doesn't care what those tasks are.

**Reply by drummerjolev** [reply]
Thank you! I'll add a `package.json` in my Django directory.

**Comment by terwer** [comment]
Thanks, it is very usefull.
